lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -495,9 +498,15 @@ unsafeGenerateKeyFromSeed
     -> Passphrase "encryption"
     -> Key depth XPrv
 unsafeGenerateKeyFromSeed (Passphrase seed, Passphrase gen) (Passphrase pwd) =
-    Key $ generateNew seed gen pwd
+    let
+        seed' = invariant
+            ("seed length in (Passphrase \"seed\") is not valid")
```

Comment:
Maybe it'll be better for debugging to indicate what size was received as part of the error message?

Reply:
refined error msg by giving actual bytestring length